viewer/packages/data-providers/src/image-360-data-providers/cdm/transformCdmAnnotations.ts

```diff
@@ -21,11 +22,15 @@ export function transformAnnotations(
     const connectedImage = image360Map[annotation.endNode.externalId][annotation.endNode.space];
 
     const properties = annotation.properties['cdf_cdm']['Cognite360ImageAnnotation/v1'];
+    assert(properties.polygon, 'Polygon must be defined in annotation properties');
 
+    const formatVersion = properties.formatVersion || '1.0.0'; // Default to '1.0.0' if not defined
```

Comment:
```suggestion
    const formatVersion = properties.formatVersion ?? '1.0.0'; // Default to '1.0.0' if not defined
```

Reply:
Applied in [00b9eb9](https://github.com/cognitedata/reveal/pull/5277/commits/00b9eb938809a330bb52a13dc379721220d5a928)